### PR TITLE
Challenge well formed strings

### PR DESCRIPTION
**Pull request by alishersadikov**
add well-formed-strings solution

### CODE REVIEW

**Comment by jcasimir** [comment]
👍

File: test/string_validator_test.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# gem 'minitest'
```
Comment (mikedao) [inline test/string_validator_test.rb:1]: You're always asserting that a thing happens. Would liked to have seen some sad path and edge case testing.

Comment (alishersadikov) [reply test/string_validator_test.rb:1]: Sure, taken into account!
